docs/sites.yml

```diff
@@ -2405,3 +2405,16 @@
     - Mobile Dev
     - React Native
   featured: false
+- title: Cameron Maske
+  main_url: 'https://www.cameronmaske.com/courses/introduction-to-pytest/'
```

Comment:
Ideally! From my understanding, the main_url generates the screenshot, and this page is a bit more interesting/custom from the homepage, so thought it would be better choice to "showcase".

Reply:
What about the `url` field then? I'd have thought that you wanted one or the other to route to your root domain!

Reply:
Good catch! Just changed it.

Reply:
hey actually it's the other way around - `url` is where screenshot is made and `main_url` is where "visit site" button takes you

Reply:
Ok! Will switch them around.